client/src/ingredients/link-list.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+function RenderLink({ link }) {
+  return (
+    <li>
+      <p>
+        <a href={link.mdn_url}>{link.short_title || link.title}</a>
+        {link.short_description && [
+          ": ",
+          <div dangerouslySetInnerHTML={{ __html: link.short_description }} />
+        ]}
+      </p>
+    </li>
+  );
+}
+
+export function LinkList({ title, links }) {
+  return (
+    <>
+      <h2 className="link-list-title">{title}</h2>
```

Comment:
I don't think you should mention "title" in the class name since it's "already there" in a sense. To make the SCSS really easy to read and debug you can change that to `h2.link-list`.
Or, even more control perhaps by putting the whole thing in a div like 
```
<div className="link-list">
   <h2>{title}</h2>
   ...
```
and then change the SCSS to:
```css
.link-list {
    h2 { border-bottom: ...}
}
```